### PR TITLE
bau/bug-fix-highest-role-switched-to-highest-role-map

### PR DESCRIPTION
**Pull request by ramsharma-prog**
This PR includes a bug fix for the User attribute "highest_role" and updates the utils version to 2.0.10, which enhances text file formatting and addresses a bug.

### CODE REVIEW

**Review by Adam-W1 — APPROVED** [review]

File: app/assess/routes.py [hunk 1]
```diff
@@ -557,7 +557,7 @@ def application(application_id):
         accounts_list=accounts_list,
         teams_flag_stats=teams_flag_stats,
         flags_list=flags_list,
-        current_user_role=g.user.highest_role,
+        current_user_role=g.user.highest_role_map,
```
Comment (tferns) [inline app/assess/routes.py:560]: This isn't used anymore, think it's better to remove it?

File: tests/test_jinja_macros.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def default_flask_g():
         full_name="Test Lead Assessor",
         email="test@example.com",
         roles=["COF_LEAD_ASSESSOR", "COF_ASSESSOR", "COF_COMMENTER"],
-        highest_role="COF_LEAD_ASSESSOR",
+        highest_role_map="COF_LEAD_ASSESSOR",
```
Comment (tferns) [inline tests/test_jinja_macros.py:44]: This should maybe be a map, but it's not used anymore so might be able to remove the g.user from this test all together?

File: tests/test_jinja_macros.py [hunk 3]
```diff
@@ -119,7 +119,7 @@ def test_criteria_macro_commenter(self, request_ctx):
             full_name="Test Commenter",
             email="test@example.com",
             roles=["COF_COMMENTER"],
-            highest_role="COF_COMMENTER",
+            highest_role_map="COF_COMMENTER",
```
Comment (tferns) [inline tests/test_jinja_macros.py:122]: This should maybe be a map, but it's not used anymore so might be able to remove the g.user from this test all together?

File: tests/test_jinja_macros.py (hunk 2 above)
Comment (ramsharma-prog) [reply tests/test_jinja_macros.py:44]: User class has an attribute "highest_role_map" in utils?

File: tests/test_jinja_macros.py (hunk 3 above)
Comment (ramsharma-prog) [reply tests/test_jinja_macros.py:122]: User class has an attribute "highest_role_map" in utils?

File: app/assess/routes.py (hunk 1 above)
Comment (tferns) [inline app/assess/routes.py]:
```suggestion
```

File: tests/test_jinja_macros.py (hunk 3 above)
Comment (ramsharma-prog) [reply tests/test_jinja_macros.py:122]: I would rather have a conversation with the person who has been working on this functionality but my focus is more on the functionality I have been working on related to mapping and formatting.

File: tests/test_jinja_macros.py (hunk 2 above)
Comment (ramsharma-prog) [reply tests/test_jinja_macros.py:44]: I would rather have a conversation with the person who has been working on this functionality but my focus is more on the functionality I have been working on related to mapping and formatting.

File: app/assess/routes.py (hunk 1 above)
Comment (ramsharma-prog) [reply app/assess/routes.py]: removed

**Review by tferns — APPROVED** [review]

**Pull request merged**